recipes/jug/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "Jug" %}
+{% set version = "1.6.4" %}
+{% set sha256 = "e739b20e7fe53ac50f5954b9e32568bdd92012dd4bd199d13e2a675ccd69d97d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like this package has a python entry point:

https://github.com/luispedro/jug/blob/master/setup.py#L69-L72

please add it as in:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
Done in update.